flan/v2/xp3_scripts/create_xp3_train_splits.py

```diff
@@ -0,0 +1,113 @@
+import json
+import csv
+# pip install -q datasets
+import datasets
+# git clone -b tr13 https://github.com/Muennighoff/promptsource.git && cd promptsource; pip install -e .
+from promptsource.templates import DatasetTemplates
```

Comment:
I'd be happy to get rid of using promptsource here. It's just used to get all the possible prompt templates for a given dataset. I'm assuming FC has that builtin already, but wasn't sure where to look.